query/queries_and_views.ipynb

```diff
@@ -854,6 +859,2063 @@
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>10592</td>\n",
+       "      <td>10392</td>\n",
+       "      <td>stopped</td>\n",
+       "      <td>http://www.nlm.nih.gov/research/umls/rxnorm</td>\n",
+       "      <td>834357</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>10594</td>\n",
+       "      <td>10392</td>\n",
+       "      <td>stopped</td>\n",
+       "      <td>http://www.nlm.nih.gov/research/umls/rxnorm</td>\n",
+       "      <td>1190795</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>29022</td>\n",
+       "      <td>26670</td>\n",
+       "      <td>stopped</td>\n",
+       "      <td>http://www.nlm.nih.gov/research/umls/rxnorm</td>\n",
+       "      <td>1804799</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>29277</td>\n",
+       "      <td>29122</td>\n",
+       "      <td>stopped</td>\n",
+       "      <td>http://www.nlm.nih.gov/research/umls/rxnorm</td>\n",
+       "      <td>834357</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>29279</td>\n",
+       "      <td>29122</td>\n",
+       "      <td>stopped</td>\n",
+       "      <td>http://www.nlm.nih.gov/research/umls/rxnorm</td>\n",
+       "      <td>1190795</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  med_req_id patient_id   status                                       system  \\\n",
+       "0      10592      10392  stopped  http://www.nlm.nih.gov/research/umls/rxnorm   \n",
+       "1      10594      10392  stopped  http://www.nlm.nih.gov/research/umls/rxnorm   \n",
+       "2      29022      26670  stopped  http://www.nlm.nih.gov/research/umls/rxnorm   \n",
+       "3      29277      29122  stopped  http://www.nlm.nih.gov/research/umls/rxnorm   \n",
+       "4      29279      29122  stopped  http://www.nlm.nih.gov/research/umls/rxnorm   \n",
+       "\n",
+       "      code practitioner_id  \n",
+       "0   834357            None  \n",
+       "1  1190795            None  \n",
+       "2  1804799            None  \n",
+       "3   834357            None  \n",
+       "4  1190795            None  "
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "with query_engine.connect() as con:\n",
+    "    con.execute(\"\"\"\n",
+    "      CREATE OR REPLACE VIEW flat_medication_request AS\n",
+    "      SELECT M.id AS med_req_id, M.subject.patientId AS patient_id, M.status,\n",
```

Comment:
For MedicationRequest, I think we also would want to have:

* intent
* statusReason
* doNotPerform